torch/_export/serde/serialize.py

```diff
@@ -2699,7 +2756,9 @@ def replace_use(a):
         n.metadata.clear()
 
     # Stage 4: Aggregate values.
-    sorted_tensor_values = dict(sorted(graph.tensor_values.items(), key=operator.itemgetter(0)))
+    sorted_tensor_values = dict(
+        sorted(graph.tensor_values.items(), key=operator.itemgetter(0))
+    )
```

Comment:
Did you change this file at all or is it all formatting?

Reply:
This is auto-formating. No change was made to this file. lol, didn't notice this file is changed so much. Let me change it back